server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/wallets/bitgo/v2/BitgoWallet.java

```diff
@@ -105,7 +111,7 @@ public String sendMany(Collection<Transfer> transfers, String cryptoCurrency, St
 
     @Override
     public String sendCoins(String destinationAddress, BigDecimal amount, String cryptoCurrency, String description) {
```

Comment:
Another possible alternative, we've explored, would be to call `BitGoCoinRequest.setFeeRate` here when the parameter is set, but this would mean having the `request` variable not to be constant.

Reply:
This approach would also be more flexible when extending the API client's request for new parameters that customers might want to set.